src/FactSet.SDK.Utils/Authentication/Constants.cs

```diff
@@ -21,10 +23,23 @@ public static class Constants
             "kty", "alg", "use", "kid", "n", "e", "d", "p", "q", "dp", "dq", "qi"
         }.ToImmutableHashSet();
 
+        public static readonly String USER_AGENT =
+            $"fds-sdk/dotnet/utils/1.0.1 ({GetOSType()}; {RuntimeInformation.FrameworkDescription})";
```

Comment:
FYI: maybe have a separate constant for this first part:
`public static readonly USER_AGENT_PREFIX = "fds-sdk/dotnet/utils/1.0.1";`